### PR TITLE
Cldc 1235 conditional shortfall

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
Tested locally, all functionality present
Minimal risk
No documentation required

ticket: https://digital.dclg.gov.uk/jira/browse/CLDC-1235

check-answers page: 
image

question page:
image

### CODE REVIEW

**Review by kosiakkatrina — COMMENTED** [review]
Do we also need to update 21/22 form? Where we do use the real form in our tests it's 21/22, although the collection window is closed for that one, maybe worth updating it just to see if there is any impact on the tests

**Comment by baarkerlounger** [comment]
@kosiakkatrina I don't think we do since the form UI itself shouldn't be used anymore for that window and the fields are staying the same here so no changes for import or export.

(But there'd be no harm in changing there also).

**Review by baarkerlounger — APPROVED** [review]

**Comment by baarkerlounger** [comment]
Actually sorry the window isn't technically closed yet (see comment at the bottom of https://digital.dclg.gov.uk/jira/browse/CLDC-1428) so we probably should do this for 21/22 form also.

**Comment by kosiakkatrina** [comment]
> @kosiakkatrina I don't think we do since the form UI itself shouldn't be used anymore for that window and the fields are staying the same here so no changes for import or export.
> 
> (But there'd be no harm in changing there also).

@baarkerlounger I think we do use the actual form for some tests, for example to check if a case log is completed, specific fields are reset based on routing etc. But perhaps no impact from this change, if that's what you mean

**Comment by natdeanlewissoftwire** [comment]
Thanks @kosiakkatrina and @baarkerlounger, have copied the changes over to 2021_2022.json as well

File: config/forms/2021_2022.json [hunk 1]
```diff
@@ -7971,14 +7956,6 @@
                 {
                   "hb": 6,
                   "hbrentshortfall": 1
-                },
-                {
```
Comment (baarkerlounger) [inline config/forms/2021_2022.json]: Why are these options being removed?

File: config/forms/2021_2022.json [hunk 2]
```diff
@@ -7876,48 +7876,33 @@
                 {
                   "hb": 6,
                   "household_charge": null
-                },
-                {
```
Comment (baarkerlounger) [inline config/forms/2021_2022.json]: Same question here

File: config/forms/2021_2022.json (hunk 1 above)
Comment (natdeanlewissoftwire) [reply config/forms/2021_2022.json]: Just fixing this now, please ignore

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**